simapp/v2/go.mod

```diff
@@ -50,6 +50,12 @@ require (
 
 require github.com/cosmos/iavl/v2 v2.0.0-alpha.4 // indirect
 
+require (
+	cosmossdk.io/server/v2/appmanager v1.0.0-beta.1
+	github.com/cometbft/cometbft/api v1.0.0
+	golang.org/x/exp v0.0.0-20241217172543-b2144cdd0a67
```

Comment:
Can we kill this import, it bitten us earlier, and so we do not want to use `x/exp` at all.

Reply:
interesting. I have removed it from the sims_runner but it is still an indirect dependency. I is used by other packages like depinject, math, orm.

Reply:
yes, that's fine, we used it there as we didn't want to bump to go 1.23. For context why we do not want to use it anymore: https://github.com/cosmos/cosmos-sdk/issues/19892#issue-2212611151 / https://github.com/cosmos/cosmos-sdk/issues/18415 / https://github.com/cosmos/cosmos-sdk/issues/19593